src/v/utils/named_type.h

```diff
@@ -99,7 +101,8 @@ class base_named_type<T, Tag, std::true_type> {
     }
 
     friend std::ostream& operator<<(std::ostream& o, const base_named_type& t) {
-        return o << "{" << t() << "}";
+        fmt::print(o, "{}", t._value);
```

Comment:
This looks like a good change, but I'm curious why you also changed the output string to no longer include the leading and trailing `{` `}`?